lib/executor/execution_test.go

```diff
@@ -18,7 +18,7 @@
  *
  */
 
-package lib
+package executor
```

Comment:
This file had to be moved, because of import loops. And in any case, we're likely to actually move the whole execution in a separate package (https://github.com/loadimpact/k6/pull/1007#discussion_r332447323), so this is probably where it belongs anyway.